docs/_markbind/layouts/default.md

```diff
@@ -0,0 +1,56 @@
+<head-bottom>
+  <link rel="stylesheet" href="{{baseUrl}}/stylesheets/main.css">
+</head-bottom>
+
+<header sticky>
+  <navbar type="dark">
+    <a slot="brand" href="{{baseUrl}}/index.html" title="Home" class="navbar-brand">AB-3</a>
+    <li><a href="{{baseUrl}}/UserGuide.html" class="nav-link">User Guide</a></li>
+    <li><a href="{{baseUrl}}/DeveloperGuide.html" class="nav-link">Developer Guide</a></li>
+    <li><a href="{{baseUrl}}/AboutUs.html" class="nav-link">About Us</a></li>
+    <li><a href="https://github.com/se-edu/addressbook-level3" target="_blank" class="nav-link"><md>:fab-github:</md></a>
+    </li>
+    <li slot="right">
+      <form class="navbar-form">
+        <searchbar :data="searchData" placeholder="Search" :on-hit="searchCallback" menu-align-right></searchbar>
+      </form>
+    </li>
+  </navbar>
+</header>
+
+<div id="flex-body">
+  <nav id="site-nav">
+    <div class="site-nav-top">
+      <div class="fw-bold mb-2" style="font-size: 1.25rem;"></div>
+    </div>
+    <div class="nav-component slim-scroll">
+      <site-nav>
+* [Home :house:]({{ baseUrl }}/index.html)
+* [User Guide]({{ baseUrl }}/UserGuide.html)
+* [Developer Guide]({{ baseUrl }}/DeveloperGuide.html)
+* [About Us]({{ baseUrl }}/AboutUs.html)
+* [Setting Up]({{ baseUrl }}/SettingUp.html)
+* [Documentation]({{ baseUrl }}/Documentation.html)
+* [Testing]({{ baseUrl }}/Testing.html)
+* [Logging]({{ baseUrl }}/Logging.html)
+* [Configuration]({{ baseUrl }}/Configuration.html)
+* [Dev Ops]({{ baseUrl }}/DevOps.html)
+      </site-nav>
+    </div>
+  </nav>
+  <div id="content-wrapper">
+    {{ content }}
+  </div>
+  <nav id="page-nav">
+    <div class="nav-component slim-scroll">
+      <page-nav />
+    </div>
+  </nav>
+</div>
+
+<footer>
+  <!-- Support MarkBind by including a link to us on your landing page! -->
+  <div class="text-center">
+    <small>[Generated by {{MarkBind}}]</small>
```

Comment:
Shall we use `Powered by` instead? Also, we can show the MarkBind icon, to give better visibility to MarkBind.
```html
<footer>
<div class="text-center">
<small>[<md>**Powered by**</md> <img src="https://markbind.org/favicon.ico" width="30"> {{MarkBind}}, generated on {{timestamp}}]</small>
</div>
</footer>
```